compiler/rustc_macros/src/newtype.rs

```diff
@@ -100,6 +103,14 @@ impl Parse for Newtype {
                     continue;
                 }
 
+                if body.lookahead1().peek(kw::ORD_IMPL) {
+                    body.parse::<kw::ORD_IMPL>()?;
+                    body.parse::<Token![=]>()?;
+                    body.parse::<kw::off>()?;
```

Comment:
For consistency with `ENCODABLE`, I think it would be better to use `custom` (since you could use this option to either write a custom `Ord` impl, or not provide an `Ord` impl at all).

Reply:
Huh. Maybe I don't understand how the `ENCODABLE = custom` works. It seems confusing that `ORD_IMPL = custom` would be used to _turn off_ ordering.